### DISCUSSION TITLE
gpt4all neovim-help boost

### DISCUSSION THREAD

**Discussion by theCollectiv** [neovim/neovim#24023] [Ideas]
It's just an idea and maybe an external plugin is better suited for this, but
...how about training gpt4all with the neovim-help* and shipping neovim with it?

* and data from frequently asked questions (like the issue section/channels/...)

**Comment by justinmk** [comment]
https://neovim.io/ now has a search engine which crawls the docs regularly.

Training a local model on the docs is an interesting idea but "talk is cheap" and it's not anything we plan to do near-term.
